### PR TITLE
add seqkit dockerfile

### PR DESCRIPTION
**Pull request by zoomzoom1011**
## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**:
* Environment:
* Test files:

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have checked my code and corrected any misspellings
- [ ] I have committed any related changes to the PR

### CODE REVIEW

**Comment by zoomzoom1011** [comment]
add seqkit dockerfile

**Review by migbro — CHANGES_REQUESTED** [review]
Please run this: https://www.notion.so/d3b/Docker-Guidelines-efdde1bc2c4e4afbb7b676c1611c9de6?pvs=4#26e2022e9a124263afc3439cf1915050 ans submit the updated config.yml. That allows circleci to find your new dockerfile and build it.

**Review by migbro — CHANGES_REQUESTED** [review]
Ok, this is a quirk that I perhaps should have mentioned. So, when you run that make config script on ubuntu, it has a drastically different sort order than on mac. That makes it impossible to track the changes introduced. If you could pull this repo locally, switch to this branch, then run the script again on your make, that'd be greatly appreciated

File: stranded:1.0/Dockerfile
Comment (dmiller15) [inline stranded:1.0/Dockerfile:23]: Conda environment is again more trouble than it's worth here. Building from an official samtools docker and adding the couple of things we need is probably the best approach.

Additionally, I'm kind of dubious about using your fork of that `how_are_we_stranded_here` repository. Seems like the issue comes down to a Python version problem that we could get around if we reverted to python3.7. I mostly just left your repo in there because the samtools docker is already 20.04 and getting python3.7 set up on that version is more of a chore than its worth.

```suggestion
FROM staphb/samtools:1.16.1

RUN apt update && apt install -y curl python-is-python3 python3-pip git

RUN python -m pip install --upgrade pip && python -m pip install git+https://github.com/zoomzoom1011/how_are_we_stranded_here.git

WORKDIR /opt
ENV KALLISTO_VERSION 0.44.0
RUN curl -sL https://github.com/pachterlab/kallisto/releases/download/v${KALLISTO_VERSION}/kallisto_linux-v${KALLISTO_VERSION}.tar.gz | tar -xz && cp kallisto_linux-v${KALLISTO_VERSION}/kallisto /usr/local/bin

WORKDIR /
COPY Dockerfile .
```

**Review by dmiller15 — CHANGES_REQUESTED** [review]

**Comment by zoomzoom1011** [comment]
@migbro  OK, I use Mac to reproduce config.yml

**Comment by migbro** [comment]
> @migbro OK, I use Mac to reproduce config.yml

Hmm, there still seems to be more than 5000 changes. @dmiller15 , can you run on your local when you get a chance and upload? Or maybe help figure out why so different, but we can also try and sort that out later to unblock this

**Comment by dmiller15** [comment]
Ok I fixed the config.

**Review by migbro — APPROVED** [review]
Looks good, passed circle ci, thanks!

**Comment by migbro** [comment]
For expediency sake, I will merge and delete branch, but for future reference @zoomzoom1011 , our policy is after 2 approvals, the person who opens the PR merges, then deletes branch.

**Pull request merged**
